### PR TITLE
Tlog block size

### PR DESCRIPTION
**Pull request by zaibon**

### CODE REVIEW

File: templates/node.zero-os/actions.py [hunk 1]
```diff
@@ -356,6 +356,10 @@ def watchdog(job):
     }
 
     async def callback(jobid, level, message, flag):
+        if "nbd" in jobid:
```
Comment (muhamadazmy) [inline templates/node.zero-os/actions.py]: Do u still need this, looks like a debug code to me

File: templates/vdisk_image/actions.py [hunk 2]
```diff
@@ -29,6 +29,7 @@ def install(job):
         cmd = "/bin/zeroctl import vdisk {vdiskid} {snapshotID} -j 20\
                --config {dialstrings} \
                --blocksize {blocksize} \
```
Comment (muhamadazmy) [inline templates/vdisk_image/actions.py]: Shouldn't the blocksize be removed ? or i miss understood Glen?

Comment (muhamadazmy) [inline templates/vdisk_image/actions.py]: Shouldn't blocksize be removed?

File: templates/node.zero-os/actions.py (hunk 1 above)
Comment (zaibon) [reply templates/node.zero-os/actions.py]: @FastGeert  ?

File: templates/vdisk_image/actions.py (hunk 2 above)
Comment (zaibon) [reply templates/vdisk_image/actions.py]: I think it should indeed since it doesn't exist on master branch of 0-disk anymore.
@FastGeert ?

**Pull request closed without merging**